test/connect.js

```diff
@@ -118,4 +118,15 @@ describe('Basic Connectivity', function() {
     }, 100 * 2);
   });
 
+
+  it('should add a new cluster server', function(done){
+    var natsServers = [uri]
+    var nc = NATS.connect({servers: natsServers});
+    nc.on('connect', function(client) {
+      client.addServer('nats://localhost:22223');
+      client.servers.length.should.equal(2);
```

Comment:
We should add another check to make sure the contents of the array are right. Should we check order?

Also, we normally randomly order the list of servers unless you ask us not to, so we need to honor that I believe, so simple push probably not sufficient.